heapsort.c

```diff
@@ -0,0 +1,61 @@
+#include<stdio.h>
+#include <conio.h>
+void Adjust(int Heap_of_Numbers[],int i)  /*Function to arrange the elements in the heap*/
+{
+int j;
+int copy;
+int Number;
+int Reference = 1;
+Number=Heap_of_Numbers[0];
+while(2*i<=Number && Reference==1)
+{
+j=2*i;   
+if(j+1<=Number && Heap_of_Numbers[j+1] > Heap_of_Numbers[j])
+j=j+1;
+if( Heap_of_Numbers[j] < Heap_of_Numbers[i])
+Reference=0;
+else
+{
+copy=Heap_of_Numbers[i];
+Heap_of_Numbers[i]=Heap_of_Numbers[j];
+Heap_of_Numbers[j]=copy;
+i=j;
+}
+}
+}
+void Make_Heap(int heap[])
+{
+int i;
+int Number_of_Elements;
+Number_of_Elements=heap[0];
+for(i=Number_of_Elements/2;i>=1;i--)
+Adjust(heap,i);
+}
+int main()
+{
+int heap[30];
+int NumberofElements;
+int i;
+int LastElement;
+int CopyVariable;

```

Comment:
Codacy found an issue: [The scope of the variable 'CopyVariable' can be reduced.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6320511)

Reply:
for swapping pupose we are using copyvariable